frontend/src/lib/components/DefinitionPopover/DefinitionPopover.scss

```diff
@@ -87,8 +89,7 @@
     }
 
     .definition-popover-timemeta {
-        display: inline;
-        margin-top: 4px;
+        margin-top: 8px;
         margin-bottom: 4px;
```

Comment:
Should these margins here and above be `rem`-s instead?

Reply:
Hm, I was about to leave them as-is, but I suppose consistency is good – so changed them all to rems